#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
uint32 gps_minutes
uint8 satellites_tracks
uint8 position_mode
uint8 velocity_mode
uint8 orientation_mode